# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: third_party/rust/mapped_hyph/Cargo.toml
# Commit: bc5b6dcb7057
# Full Hash: bc5b6dcb7057cf610efe29e14b65e9c3aa34d1c3
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2020-10-12 15:27:19
# Description:
#   Bug 1669162 - Update mapped_hyph to 0.4.2 so that .dic parse errors are non-fatal. r=heycam
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D92435
# ==============================================================================

diff -r a083e04138aa -r bc5b6dcb7057 third_party/rust/mapped_hyph/Cargo.toml
--- a/third_party/rust/mapped_hyph/Cargo.toml	Mon Oct 12 09:52:22 2020 +0000
+++ b/third_party/rust/mapped_hyph/Cargo.toml	Mon Oct 12 10:23:27 2020 +0000
@@ -1,7 +1,7 @@
 [package]
 name = "mapped_hyph"
 description = "Hyphenation using precompiled memory-mapped tables"
-version = "0.4.0"
+version = "0.4.2"
 authors = ["Jonathan Kew <jfkthame@gmail.com>"]
 license = "MIT/Apache-2.0"
 edition = "2018"
@@ -9,6 +9,8 @@
 [dependencies]
 memmap = "0.7.0"
 arrayref = "0.3.5"
+log = "0.4"
+env_logger = "0.7.1"
 
 [dev-dependencies]
 criterion = "0.3"